manage_custom_field_edit_page.php

```diff
@@ -108,6 +108,22 @@
 			</select>
 		</td>
 	</tr>
+  <tr>
+      <td class="category">
+          <?php echo lang_get( 'custom_field_color_foreground' ) ?>
+      </td>
+      <td>
+          <input type="color" id="custom-field-color-foreground" name="color_foreground" size="8" value="<?php echo $t_definition['color_foreground'] != '' ? $t_definition['color_foreground'] : '#000000' ?>" />
```

Comment:
Didn't try, but seems that color input type is not supported in IE, which is still supported in Mantis until now.
https://caniuse.com/#feat=input-color

Maybe you can try, or I will try later.
I think it's ok if there is a text input field instead of a color picker in IE.

Reply:
Correct. Color input doesn't work in IE.
Removed input type color. Replaced by plain text input.